holoviews/util/__init__.py

```diff
@@ -215,21 +215,25 @@ def _cellmagic(cls, options, obj, strict=False):
             return StoreOptions.set_options(obj, options)
 
     @classmethod
-    def _linemagic(cls, options, strict=False):
+    def _linemagic(cls, options, strict=False, backend=None):
         "Deprecated, not expected to be used by any current code"
-        options, failure = cls._process_magic(options, strict)
+        backends = None if backend is None else [backend]
+        options, failure = cls._process_magic(options, strict, backends=backends)
         if failure: return
         with options_policy(skip_invalid=True, warn_on_skip=False):
-            StoreOptions.apply_customizations(options, Store.options())
+            StoreOptions.apply_customizations(options, Store.options(backend=backend))
 
 
     @classmethod
-    def defaults(cls, *options):
+    def defaults(cls, *options, **kwargs):
         """
         Set default options for a session, whether in a Python script or
         a Jupyter notebook.
         """
```

Comment:
Would be nice to have a Google style docstring here documenting the backend kwarg.